src/main/java/me/partlysanestudios/partlysaneskies/OneConfigScreen.java

```diff
@@ -421,11 +421,31 @@ public void resetBrokenStrings() {
     @Switch(
         name = "Toggle Run Colors in Partymanager",
         subcategory = "Party Manager",
-        description = "Toggles the colors of the runs in the party manager, Red = 0 Runs, Yellow 1-9 Runs, Green 10+ Runs",
+        description = "Toggles the colors of the runs in the party manager. ",
         category = "Dungeons"
     )
     public boolean toggleRunColors = true;
 
+    @Number(
+        name = "Customize Max Runs for Red in Run Colors",
+        min = 0,
+        max = 2147483647, //Hopefully that doenst break anything
+        subcategory = "Party Manager",
+        description = "Customize maximum runs required for the color red",
+        category = "Dungeons"
+    )
+    public int runColorsRedMax = 1;
+
+    @Number(
+        name = "Customize Max Runs for Yellow in Run Colors",
+        min = 0,
+        max = 2147483647, //Hopefully that doenst break anything
```

Comment:
Replace with Integer.MAX_VALUE, other then that it looks good and I'll merge it in

Reply:
ooh thats neat